content/MBC1.md

```diff
@@ -8,30 +8,30 @@ types of MBCs.
 Note that the memory in range 0000-7FFF is used both for reading from
 ROM and writing to the MBCs Control Registers.
 
-## ROM/RAM access
+## Memory
 
 ### 0000-3FFF - ROM Bank 00/20/40/60 (Read Only)
 
-This area normally contains the first 16KBytes (bank 00) of the cartridge
-ROM. Can contain banks 20/40/60 in mode 1 (see below), or banks 10/20/30
+This area normally contains the first 16 KiB (bank 00) of the cartridge
+ROM. Can contain banks $20/$40/$60 in mode 1 (see below), or banks $10/$20/$30
 in mode 1 for a 1MB MBC1 multi-cart (see below).
 
 ### 4000-7FFF - ROM Bank 01-7F (Read Only)
 
-This area may contain any of the further 16KByte banks of the ROM. Cannot
-address any banks where the main ROM banking register would be 00h, which
+This area may contain any of the further 16 KiB banks of the ROM. Cannot
+address any banks where the main ROM banking register would be $00, which
 usually means banks 00/20/40/60. Instead, it automatically maps to 1 bank
 higher (01/21/41/61).
 
 ### A000-BFFF - RAM Bank 00-03, if any (Read/Write)
 
 This area is used to address external RAM in the cartridge (if any).
 External RAM is often battery-backed, allowing for the storage of game data while the Game Boy is turned off, or if the
-cartridge is removed from the Game Boy. Available RAM sizes are: 2KByte
-(at A000-A7FF), 8KByte (at A000-BFFF) and 32KByte (in form of four 8K
+cartridge is removed from the Game Boy. Available RAM sizes are: 2 KiB
+(at A000-A7FF), 8 KiB (at A000-BFFF) and 32 KiB (in form of four 8K
```

Comment:
$ for the hex here and in the line before, please!

Reply:
Fixed